app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="text_header">
+        <item name="android:layout_marginTop">@dimen/margin_xlarge</item>
+        <item name="android:layout_marginStart">@dimen/margin_xlarge</item>
+        <item name="android:layout_marginEnd">@dimen/margin_xlarge</item>
+        <item name="android:fontFamily">@font/montserrat_bold</item>
+        <item name="android:textColor">@color/font_black</item>
+        <item name="android:textSize">@dimen/text_xlarge</item>
+    </style>
+    <style name="button_primary_big">
```

Comment:
Styling names start usually with Uppercase and do not contain _ . ButtonPrimaryBig or Button.Primary.Big but for using the . you need other styling for Button and other styling for Button.Primary.